pkg/operator/cleanup/cleanup_controller_test.go

```diff
@@ -123,7 +123,8 @@ func TestStaleCredentialsRequestReconcile(t *testing.T) {
 				Client: fakeClient,
 			}
 
-			_, err := rscr.Reconcile(reconcile.Request{
+			ctx := context.TODO()
+			_, err := rscr.Reconcile(ctx, reconcile.Request{
```

Comment:
Any reason `context.TODO()` isn't inlined here, like "normal"?

Reply:
No particular reason. Fixed and pushed.